rails_event_store/lib/rails_event_store/async_dispatcher.rb

```diff
@@ -1,27 +1,25 @@
 module RailsEventStore
```

Comment:
`ActiveJobDispatcher` is `RailsEventStore::`, however `AsyncDispatcher` can be moved to `RubyEventStore::` and be used in non-rails scenarios.

Reply:
We still use there `AsyncProxyStrategy::AfterCommit` which uses `ActiveRecord` but maybe this should stay in `RailsEventStore::` and only `AsyncProxyStrategy::Inline` should moved with `AsyncDispatcher` to `RubyEventStore::`

Reply:
:+1: